src/backend/drm/surface/atomic.rs

```diff
@@ -570,7 +570,7 @@ impl AtomicDrmSurface {
             } else {
                 if current.mode != pending.mode {
                     if let Err(err) = self.fd.destroy_property_blob(current.blob.into()) {
-                        warn!("Failed to destory old mode property blob: {}", err);
+                        debug!("Failed to destory old mode property blob: {}", err);
```

Comment:
Well this error only occurs at launch, because the current_mode is not an `Option`, but initialized with a bogus null-blob (which obviously fails to be destroyed). Any further occurrences would indicate an actual error.

However without making it an `Option` and littering the code with `unwraps`, I don't see a good way to mitigate that first log message.

Reply:
Wouldn't it be possible to just check for `Unknown(0)` or to have a flag on `current.blob` which stores if the blob has been initialized or not?

Attempting this operation when it is know that it will fail seems like it's not ideal anyway, so maybe we can just skip the destruction entirely?